content/STRUCTURE.json

```diff
@@ -270,6 +285,16 @@
      "generator": "link",
      "meta": {"url": "https://github.com/lxc/lxcfs"}},
 
+    {"path": "/lxcfs/github-actions/",
+     "menu": ["LXCFS", "GitHub Actions"],
+     "generator": "link",
+     "meta": {"url": "https://github.com/lxc/lxcfs/actions"}},
+
+    {"path": "/lxcfs/jenkins-ci/",
+     "menu": ["LXCFS", "Jenkins CI"],
+     "generator": "link",
+     "meta": {"url": "https://jenkins.linuxcontainers.org/view/LXCFS/"}},
```

Comment:
This should be dropped, there is no meaningful CI tests done for LXCFS on Jenkins, it's just basic commit signature stuff.